libopenage/gamestate/player.h

```diff
@@ -29,10 +29,10 @@ class Player {
 
 	// players can't be copied to prevent duplicate IDs
 	Player(const Player &) = delete;
-	Player(Player &&) = default;
+	Player(Player &&) = delete;
```

Comment:
Because the player's `id` member was defined as `const` which implictly deleted move constructors. I've now added a protected copy constructor and enabled moving just like for `GameEntity`, so this should work much better.